dockerfiles/Dockerfile.init

```diff
@@ -1,2 +1,2 @@
-FROM alpine:3
-RUN apk add --no-cache iptables
+FROM alpine:3.17.2
+RUN apk update && apk add --no-cache iptables "libcrypto3>=3.0.8-r1" "libssl3>=3.0.8-r1"      
```

Comment:
Could we add a note to the PR description explaining this change?

Reply:
This was because of a CVE that was detected. Patched version should be out [today](https://github.com/alpinelinux/docker-alpine/issues/312#issuecomment-1488803235)

Reply:
Do we want to wait to include this patched version? @keithmattix

Reply:
@jaellio [this ](https://github.com/openservicemesh/osm/actions/runs/4536753241/jobs/7993820813#step:5:103) is the error we got

Reply:
Yes please do; it's working on my PR

Reply:
This update can be addressed in Keith's PR. 

LGTM